main_test.go

```diff
@@ -26,6 +26,7 @@ var mockConfig = config{
 	SectionSubtitle:              "Commit message",
 	SectionText:                  "Commit message body",
 	SectionHeaderImage:           "",
+	SectionImage:                 "",
```

Comment:
This will need some sort of input rather than an empty `string` if you intend to test whether the image section contains an image `struct`.